### PR TITLE
Enable DotNet SDK Track2 for SDK Automation

### PR DESCRIPTION
**Pull request by PhoenixHe-NV**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specificationRepositoryConfiguration.json [hunk 1]
```diff
@@ -19,6 +19,11 @@
       "integrationRepository": "AzureSDKAutomation/azure-sdk-for-net",
       "mainRepository": "Azure/azure-sdk-for-net"
     },
+    "azure-sdk-for-net-track2": {
```
Comment (weshaggard) [inline specificationRepositoryConfiguration.json:22]: I'm not sure I like calling this track 2 could we instead have it mirror the version of the code generator?

Comment (NullMDR) [reply specificationRepositoryConfiguration.json:22]: IMO you have another set of configuration for .NET track2 here so I would like to make it another "language" for SDK Automation

**Comment by JackTn** [comment]
Hi, @PhoenixHe-TSP. The PR has be closed for a long time and it's related branch still exist. If no further used for over 14 days, I will delete this related branch. Please tell me if you still need this branch.

**Comment by JackTn** [comment]
Hi, @PhoenixHe-TSP. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Pull request closed without merging**